content/en/dora_metrics/setup/_index.md

```diff
@@ -11,25 +15,56 @@ The DORA Metrics private beta is closed. Fill out the form below to be added to
 
 ## Overview
```

Comment:
I think we should extend the overview to mention the APM Deployment tracking and Pagerduty integrations.